jest.config.js

```diff
@@ -18,4 +18,5 @@ module.exports = {
   moduleDirectories: ["node_modules", "src"],
   globals: {"__DEV__": false},
   collectCoverage: true,
+  coveragePathIgnorePatterns: ['www/js/services/commHelper.ts']
```

Comment:
Great, this seems to work!!

Can we add a note here for explanation?

```suggestion
  // several functions in commHelper do not have unit tests; see note in commHelper.test.ts
  coveragePathIgnorePatterns: ['www/js/services/commHelper.ts'],
```